include/RAJA/policy/cuda/teams.hpp

```diff
@@ -46,6 +42,7 @@ __global__ void launch_global_fcn(LaunchContext ctx, BODY body_in)
 
 template <bool async>
 struct LaunchExecute<RAJA::expt::cuda_launch_t<async, 0>> {
+
   template <typename BODY_IN>
   static void exec(LaunchContext const &ctx, BODY_IN &&body_in)
```

Comment:
Can this be removed or implemented with the the resource version?

Reply:
I think we can, let me see what I can do.

Reply:
I think it can, but I'll make the change after the release along with API rework.